ForumProject/notifications/serializers.py

```diff
@@ -1,9 +1,18 @@
 from rest_framework import serializers
-from .models import Notification
+from .models import Notification, NotificationPreferences
 
 
 class NotificationSerializer(serializers.ModelSerializer):
     class Meta:
         model = Notification
         fields = ['project', 'startup', 'investor', 'trigger', 'initiator', 'date_time']
         read_only_fields = ['project', 'startup', 'investor', 'trigger', 'initiator', 'date_time']
+
+class NotificationPreferencesSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = NotificationPreferences
+        fields = ['email_on_followers_change',
+            'email_on_share_subscription',
+            'in_app_on_followers_change',
+            'in_app_on_share_subscription'
+            ]
```

Comment:
Your serializer does not include the startup field. Depending on the application's requirements, you might want to expose the startup relationship to allow read or write operations through the API. This would be especially important if the client needs to know which startup the preferences are associated with or if they need to set preferences for a specific startup.

Reply:
Done. Now the client gets startup_id with the response.